src/api/plugins/video/warren_video/models.py

```diff
@@ -5,15 +5,15 @@
 from warren.fields import Date
 
 
-class VideoDayViews(BaseModel):
-    """Model to represent video views for a date."""
+class VideoDayStats(BaseModel):
```

Comment:
Should it be Video specific or in warren's core?

```suggestion
class DayCount(BaseModel):
```

Reply:
In my current proposal, these classes are currently only used in the video package. Therefore, it's preferable to keep the code close to its usage. Thus, having an explicit naming is desirable.

Reply:
However, as I would refactor the indicator in a shared one, I might need to extract them in warren's core.

Reply:
I have changed my mind, and followed your recommendation. I have refactored these models in warren's core, and renamed them to `DailyCounts` and `DailyCount`.